### DISCUSSION TITLE
About image processing

### DISCUSSION THREAD

**Discussion by CheungBH** [open-mmlab/mmpose#860] [General]
Hello. Thanks for your excellent job. When I was doing pose estimation research, I found that there are several methods to unify the different box size into a fixed input-size. One is resizing directly, and another uses letterbox method, that the image will be padded by gray color,

I wonder which method do you use, and are there any references? In my opinion, using padding is better, but I don't know whether I am right.

**Comment by jin-s13** [comment]
We use the center/scale to crop and resize the image into desired input-size, see `TopDownAffine`.

**Comment by CheungBH** [comment]
Thank you. I also notice that the gaussian heatmap of samples in mmpose is circular. But I also notice that there are some repo will restrict the gaussian within a square (like 7x7). I wonder whether there are some reference about it?

<img width="577" alt="3f5252ad525f83c10c08d0d6baf442a" src="https://user-images.githubusercontent.com/39186017/113473415-ebb0b400-949b-11eb-8237-f8bcd763d70f.png">

**Comment by jin-s13** [comment]
For higher speed, most repos (including mmpose) restrict the gaussian within a fixed range (3-sigma).
